sdk/core/ts-http-runtime/review/ts-http-runtime-internal-policies-browser.api.diff.md

```diff
@@ -0,0 +1,175 @@
+# API Report Diff for browser runtime
+
+This file contains only the differences from the Node.js API.
+For the complete API surface, see the corresponding -node.api.md file.
+
+```
+- export function decompressResponsePolicy(): PipelinePolicy;
++ export function decompressResponsePolicy(): never;
+- // @public
++ // @public (undocumented)
+- // @public @deprecated
++ // @public (undocumented)
+- export function getDefaultProxySettings(proxyUrl?: string): ProxySettings | undefined;
++ export function getDefaultProxySettings(): never;
+- export function proxyPolicy(proxySettings?: ProxySettings, options?: {
++ export function proxyPolicy(): never;
+-     customNoProxyList?: string[];
++ 
+- }): PipelinePolicy;
```

Comment:
Since this definition spans multiple lines in the Node version and not the browser version, the diff from here downwards is just noise. Any other differences between runtimes which appear below this point will get caught in the noise which I think really limits the value of the report.

Reply:
It doesn't even need to be semantic, it just needs to do more than compare line by line. The current approach is problematic when one side has a line added somewhere in the middle of the file but no differences otherwise -- with the current algorithm, every line below the added line gets marked as a difference when it really shouldn't be.

Maybe the diff calculation can be outsourced to an existing tool/library (git diff maybe somehow? I don't know)